alcs-frontend/src/app/features/home/audit/audit-subtasks.component.scss

```diff
@@ -0,0 +1,54 @@
+@import '../../../../styles/colors';
+
+.assignee {
+  margin: 18px 18px 18px 0;
```

Comment:
I fixed this in my MR for gis component, but I think it should just be margin: 18px 0; so it lines up with the edges.